exercises/07-normalizr/backend/server.ts

```diff
@@ -0,0 +1,77 @@
+import express from 'express'
+
+import { User, UserData } from '../src/modules/users/user-types'
+
+const port = 3001
+
+const areStringsSame = (a: string, b: string): boolean =>
```

Comment:
I'd go for "equal" instead of "same". I'd also consider names like caseInsensitiveCompare (https://developer.apple.com/documentation/foundation/nsstring/1414769-caseinsensitivecompare), caseInsensitiveEqual, areCaseInsensitiveEqual, areEqual

Reply:
renamed to `caseInsensitiveEqual`, see https://github.com/salsita/react-training/pull/59/commits/c7b960b599747f88aebd2d95c3e13abc81385778